LICENSE

```diff
@@ -1,16 +1,16 @@
-Limerick Permissive License (LPL v2)
-
-(c) 2022 Alexander E Genaud
-
-Permission is granted hereby,
-to copy, share, use, modify,
-   for purposes any,
-   for free or for money,
-provided these rhymes catch eye.
-
-This work "as is" I provide,
-no warranty express or implied,
-   for, no purpose fit,
-   'tis unmerchantable shit.
-Liability for damages denied.
-
+ Limerick Permissive License (LPL v2)
+ 
+ (c) 2022 Alexander E Genaud
+ 
+ Permission is granted hereby,
+ to copy, share, use, modify,
+    for purposes any,
+    for free or for money,
```

Comment:
```suggestion
    for free or some money,
```

- "some" has same one syllable as "for"
- reduce the number of "for" from three to two
- explicitly implies (explies?) "some" rather than "huge cost"
- the "s" in "some money" may parallel "purposes any" better?
- does not change meaning, yet hints toward generosity.